packages/mdc-button/README.md

```diff
@@ -137,6 +137,25 @@ Disabled buttons cannot be interacted with and have no visual interaction effect
 </button>
 ```
 
+## Additional Information
+
+### Accessibility
+
+Material Design spec advises that touch targets should be at least 48 x 48 px.
```

Comment:
Buttons will always have at least 48px width since we add 24px padding-left/right, so this is still valid. Going to keep this to consistent with Material spec, to avoid any confusion.